### PR TITLE
docs: self-introduction

### PR DESCRIPTION
**Pull request by JayoonKoo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,9 +3,22 @@
 <head>
 	<meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>My First HTML Page with git</title>
```
Comment (ghost) [inline index.html:6]: Could you change title to 'My Best HTML Page with git' ?

File: index.html [hunk 2]
```diff
@@ -3,9 +3,22 @@
...
+	<title>Introduce my self</title>
 </head>
 <body>
-	
+	<div>
+		<h1>간단한 자기 소개</h1>
```
Comment (ghost) [inline index.html]: Change Tag from h1 to h2

**Review by ghost — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (JayoonKoo) [reply index.html:6]: yes

File: index.html (hunk 2 above)
Comment (JayoonKoo) [reply index.html]: yes

**Review by ghost — APPROVED** [review]
good job

**Pull request merged**